discovery-handler-modules/debug-echo-discovery-handler/src/main.rs

```diff
@@ -0,0 +1,49 @@
+use akri_debug_echo::{
+    discovery_handler::{DiscoveryHandler, DISCOVERY_PORT},
+    get_register_request,
+};
+use akri_discovery_utils::{
+    discovery::{server::run_discovery_server, DISCOVERY_HANDLER_PATH},
+    registration_client::{register, register_again},
+};
+use log::{info, trace};
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
+    env_logger::try_init()?;
+    info!("main - debugEcho discovery handler started");
+    // Determine whether to serve discovery handler over UDS or IP based on existence
+    // of the environment variable POD_IP.
+    let mut use_uds = true;
+    let mut endpoint: String = match std::env::var("POD_IP") {
```

Comment:
All of Akri's supported Discovery Handlers support running serving their discovery service over IP. Should this support be removed from the Discovery Handlers? It may just crowd it out and will probably not be used. It does provide an example for how one may configure their discovery handler to run over IP.